package.json

```diff
@@ -79,7 +79,7 @@
     ]
   },
   "dependencies": {
-    "@typescript-eslint/experimental-utils": "^4.11.1",
+    "@typescript-eslint/experimental-utils": "^5.0.0",
```

Comment:
```suggestion
    "@typescript-eslint/utils": "^5.10.0",
```

?

😀

Reply:
@G-Rath any news on this one? This change is what's keeping this around: https://github.com/jest-community/eslint-plugin-jest/blob/78a714187d1ab0fffd2b8dc5f1ea5e41692856e6/package.json#L158-L160

Reply:
ah right, in that case let's axe this from `eslint-plugin-jest` - it's not possible to support ESLint v8 because it removes the synchronous API this plugin requires and I can't use the async version since rules can't be async (yet) 🤷 

(I thought I put that somewhere, but can't find the comment)